src/pages/players/[slug].tsx

```diff
@@ -37,7 +37,7 @@ const PlayerPage = ({ data, errorMessage, errorStatus }: TSsrRole<TData>) => {
   const { t } = useTranslation()
   const router = useRouter()
 
-  const { isAdmin, player } = data as TData
+  const { isAdmin, player } = (data as TData) || {}
```

Comment:
```suggestion
  const { isAdmin, player } = data || {}
```

You don't need to cast this type anymore. In fact, casting it was a mistake in the first place, there's a reason why typescript complains about it - data can be null.

Reply:
okay now I've replicated it😅
`|| {}` helped with this:
![image](https://user-images.githubusercontent.com/80927085/198847415-3ab1d0f2-6cf3-4b18-8ddd-76b90e606565.png)
so we could show this:
![image](https://user-images.githubusercontent.com/80927085/198847403-781d8d0f-bf3e-4930-b59c-de5551e93807.png)

and the primary problem was fixed in handle slug backend pr that you already saw🙏

Reply:
Oh, ok. Got it